GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifyAccountDetail.m

```diff
@@ -16,5 +16,34 @@
 
 #import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifyAccountDetail.h"
 
+#import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiableAccountDetail.h"
+#import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiedAccountDetailResult.h"
+
 @implementation GIDVerifyAccountDetail
+
+#if TARGET_OS_IOS || TARGET_OS_MACCATALYST
+
+- (void)verifyAccountDetails:(NSArray<GIDVerifiableAccountDetail *> *)accountDetails
+    presentingViewController:(UIViewController *)presentingViewController
+                  completion:(nullable void (^)(GIDVerifiedAccountDetailResult *_Nullable verifyResult, NSError *_Nullable error))completion {
+    // TODO: Implement this method.
```

Comment:
In general, it's a good practice to link TODOs to an issue on the repo. I know we have internal tracking for this work, but we can talk about creating visibility here as well.

Reply:
Sounds good! I created the [issue 383](https://github.com/google/GoogleSignIn-iOS/issues/383) and added the issue number to the TODO. Let me know if there's a better way of linking the issue or if I should be adding in the entire link.